src/Client/Views/BaseView.fs

```diff
@@ -27,6 +27,7 @@ let createNavigationTab (pageLink: Routing.Page) (model:Model) (dispatch:Msg-> u
             ]
         ] [
             Text.span [] [
+                /// does not work for me in excel, and in excel online i have a fixed width that always shows the icon
```

Comment:
Please open an issue if this does not work anymore

Reply:
i noticed it updates everytime i switch the subpage, but we should switch to css:

`@media only screen and (max-width: XXXpx) {
 previousClass: {style}
}
`

Reply:
Yes that is way cleaner. Can you open an issue for that so we have it in the backlog?